### DISCUSSION TITLE
webrtc?

### DISCUSSION THREAD

**Discussion by golfymate** [xjasonlyu/tun2socks#98] [Ideas]
is it possible to put a -flag or tweak for custom webrtc?

just like we do with --dns-servers:

**Comment by xjasonlyu** [comment]
More context pls

**Reply by golfymate** [reply]
Is it possible to spoof our WebRTC IP?
The goal is to match the Proxy IP instead of using the default local IPv4 as webRTC - Example `10.0.0.1`

https://browserleaks.com/ip/ 

> WebRTC Leak Test
> --
> Local IP address | n/a
> Public IP address |  [PROXY-IP-HERE]
